AIO/src/org/aio/activities/banking/ItemReqBanking.java

```diff
@@ -164,6 +166,8 @@ public int execute() throws InterruptedException {
                         setFinished();
                     } else if (getBank().isOpen()) {
                         getBank().close();
+                    } else if (getBank().getWithdrawMode() != org.osbot.rs07.api.Bank.BankMode.WITHDRAW_ITEM) {
```

Comment:
The bank should be closed at this point, so we don't want to try to change the withdraw mode.